### ISSUE TITLE
Text Scaling on Home Page

### ISSUE THREAD

**Issue by shiwal25** [OneBusAway/onebusaway-android#1416]
The TextViews use dp (density-independent pixels). For accessibility, sp (scale-independent pixels) should be used to improve readability for users with vision impairments.

Screenshot:

![Image](https://github.com/user-attachments/assets/0c9446d4-b6cf-4b3b-8a04-361e04f3f672)

**Comment by Shiwal25** [comment]
@amrhossamdev assign me the issue

**Comment by jenniferer2** [comment]
@aaronbrethorst can you assign me the issue I came up with a solution

**Comment by shiwal25** [comment]
@jenniferer2 it is already resolved.
